netbox_prometheus.py

```diff
@@ -134,6 +134,9 @@ def write_metrics(self, filename):
     #METRICS = "/tmp/netbox.prom"
 
     nb = pynetbox.api(API_URL, token=API_TOKEN)
+    # Wether or not to validate the TLS certificate of API_URL
+    nb.http_session.verify = True
```

Comment:
AFAICS, this is true by default (which is what I'd expect):

```
root@prometheus:~# python3 -i nb
>>> nb.http_session
<requests.sessions.Session object at 0x7f072cbf9160>
>>> nb.http_session.verify
True
```

However, I guess it's useful to have it as documentation on how to disable HTTPS verification if required.

Reply:
Hello, we had a bit of an issue when we started playing with this with one of our internally signed certificates. Python request just did not like it so I was digging around to figure out how to disable it. So I found that. The plan was to set another ENV var and set it to true by default for us to disable but I couldn't get it to work. So I just left it True in the commit so I could toggle it off in our internal environment. Getting an ENV variable such as SSL_VALIDATE would be ideal.